lib/SILOptimizer/Transforms/SimplifyCFG.cpp

```diff
@@ -231,7 +232,8 @@ bool SimplifyCFG::threadEdge(const ThreadInfo &ti) {
         Builder.createBranch(SEI->getLoc(), ThreadedSuccessorBlock, {UED});
       } else {
         assert(SEI->getDefaultBB() == ThreadedSuccessorBlock);
-        Builder.createBranch(SEI->getLoc(), ThreadedSuccessorBlock, SEI->getOperand());
+        Builder.createBranch(SEI->getLoc(), ThreadedSuccessorBlock,
+                             SEI->getOperand());
```

Comment:
Since phis mean something in OSSA more than simply a branch argument. We should either change this implementation to RAUW without creating a phi. Or explicitly initialize a new phi.

Reply:
It looks like this commit fixed the places where we were accidentally creating new phis without properly creating the phi:
https://github.com/apple/swift/pull/62959/commits/9311e948a55aa548a38c15733d34620b949384c7